tests/devices/qutrit_mixed/test_qutrit_mixed_simulate.py

```diff
@@ -0,0 +1,763 @@
+    if subspace == (0, 1):
+        return np.array([-np.sin(phi), np.cos(phi), 0, np.sqrt(1 / 3)])
+    if subspace == (0, 2):
+        return np.array(
+            [
+                0,
+                np.cos(phi / 2) ** 2,
+                -np.sin(phi),
+                np.sqrt(1 / 3) * (np.cos(phi) - np.sin(phi / 2) ** 2),
+            ]
+        )
+    pytest.skip(f"Test cases doesn't support subspace {subspace}")
+
+
+def expected_TRX_circ_expval_jacobians(phi, subspace):
+    """Find the Jacobians of expect-values of GellManns 2,3,5,8
+    on a circuit with a TRX gate on subspace (0,1) or (0,2)"""
+    if subspace == (0, 1):
+        return np.array([-np.cos(phi), -np.sin(phi), 0, 0])
+    if subspace == (0, 2):
+        return np.array([0, -np.sin(phi) / 2, -np.cos(phi), np.sqrt(1 / 3) * -(1.5 * np.sin(phi))])
+    pytest.skip(f"Test cases doesn't support subspace {subspace}")
+
+
+def expected_TRX_circ_state(phi, subspace):
+    """Find the state after applying TRX gate on |0>"""
+    expected_vector = np.array([0, 0, 0], dtype=complex)
+    expected_vector[subspace[0]] = np.cos(phi / 2)
+    expected_vector[subspace[1]] = -1j * np.sin(phi / 2)
+    return np.outer(expected_vector, np.conj(expected_vector))
+
+
+class TestCurrentlyUnsupportedCases:
+    """Test currently unsupported cases, such as sampling counts or samples without shots, or probs with shots"""
+
+    # pylint: disable=too-few-public-methods
+    def test_sample_based_observable(self):
+        """Test sample-only measurements raise a NotImplementedError."""
+
+        qs = qml.tape.QuantumScript(measurements=[qml.sample(wires=0)])
+        with pytest.raises(NotImplementedError):
+            simulate(qs)
+
+    @pytest.mark.parametrize("mp", [qml.probs(0), qml.probs(op=qml.GellMann(0, 2))])
+    def test_invalid_samples(self, mp):
+        """Test Sampling MeasurementProcesses that are currently unsupported on this device"""
+        qs = qml.tape.QuantumScript(ops=[qml.TAdd(wires=(0, 1))], measurements=[mp], shots=10)
+        with pytest.raises(NotImplementedError):
+            simulate(qs)
+
+
+def test_custom_operation():
+    """Test execution works with a manually defined operator if it has a matrix."""
+
+    # pylint: disable=too-few-public-methods
+    class MyOperator(qml.operation.Operator):
+        num_wires = 1
+
+        @staticmethod
+        def compute_matrix():
+            return np.array([[0, 0, 1], [0, 1, 0], [1, 0, 0]])
+
+    qs = qml.tape.QuantumScript([MyOperator(0)], [qml.expval(qml.GellMann(0, 8))])
+
+    result = simulate(qs)
+    assert qml.math.allclose(result, -np.sqrt(4 / 3))
+
+
+@pytest.mark.tf
+@pytest.mark.parametrize("op", [qml.TRX(np.pi, 0), qml.QutritBasisState([1], 0)])
+@pytest.mark.parametrize("interface", ("jax", "tensorflow", "torch", "autograd", "numpy"))
+def test_result_has_correct_interface(op, interface):
+    """Test that even if no interface parameters are given, result is correct."""
```

Comment:
```suggestion
@pytest.mark.all_interfaces
@pytest.mark.parametrize("op", [qml.TRX(np.pi, 0), qml.QutritBasisState([1], 0)])
@pytest.mark.parametrize("interface", ("jax", "tensorflow", "torch", "autograd", "numpy"))
def test_result_has_correct_interface(op, interface):
    """Test that even if no interface parameters are given, result is correct."""
```

Reply:
Good catch, thank you I forgot to change that back when I was testing it.